test/tests/postprocessors/misc_pps/misc_pps_test.i

```diff
@@ -14,16 +14,13 @@
   [./u]
     order = SECOND
     family = LAGRANGE
-    [./InitialCondition]
-      type = ConstantIC
-      value = 0
-    [../]
   [../]
 []
 
 [AuxVariables]
   [./aux_var]
     family = SCALAR
+    initial_condition = 42
```

Comment:
Ok, I thought you were just removing code in this patch, so it looked like a left over from something else...